engine/userguide/storagedriver/aufs-driver.md

```diff
@@ -4,248 +4,222 @@ keywords: 'container, storage, driver, AUFS '
 title: Docker and AUFS in practice
 ---
 
-AUFS was the first storage driver in use with Docker. As a result, it has a
-long and close history with Docker, is very stable, has a lot of real-world
-deployments, and has strong community support. AUFS has several features that
-make it a good choice for Docker. These features enable:
+AUFS a *union filesystem*. The `aufs` storage driver is the default storage
+driver used for managing images and layers on Docker, for Ubuntu and Debian.
```

Comment:
- for Debian, I don't think it's supported anymore in their newer kernels @justincormack ?
- for Ubuntu 16.04, and 14.04 with the newer 4.x kernels installed (which I _think_ is now the default on some installations with 14.04.2), overlay2 is the default